styles/namemc/catppuccin.user.css

```diff
@@ -0,0 +1,582 @@
+    }
+    .btn-secondary {
+      #bsButton(@text, @overlay0);
+    }
+    .btn-success {
+      #bsButton(@crust, @green);
+    }
+    .btn-info {
+      #bsButton(@crust, @sapphire);
+    }
+    .btn-warning {
+      #bsButton(@crust, @yellow);
+    }
+    .btn-danger {
+      #bsButton(@crust, @red);
+    }
+    .btn-light {
+      #bsButton(@crust, @overlay2);
+    }
+    .btn-dark {
+      #bsButton(@text, @surface0);
+    }
+
+    .btn-outline-primary {
+      #bsButtonOutline(@accent-color, @text);
+    }
+    .btn-outline-secondary {
+      #bsButtonOutline(@overlay0, @text);
+    }
+    .btn-outline-success {
+      #bsButtonOutline(@green, @text);
+    }
+    .btn-outline-info {
+      #bsButtonOutline(@sapphire, @crust);
+    }
+    .btn-outline-warning {
+      #bsButtonOutline(@yellow, @crust);
+    }
+    .btn-outline-danger {
+      #bsButtonOutline(@red, @text);
+    }
+    .btn-outline-light {
+      #bsButtonOutline(@text, @crust);
+    }
+    .btn-outline-dark {
+      #bsButtonOutline(@surface0, @text);
+    }
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    .text-success {
+      color: @green !important;
+    }
+
+    .btn-link {
+      --bs-btn-disabled-color: @overlay0;
+      --bs-btn-disabled-border-color: transparent;
+      --bs-btn-focus-shadow-rgb: @blue;
+    }
+
+    #header {
+      form {
+        div.input-group.input-group-lg {
+          input.form-control {
+            background-color: @base;
+
+            &:focus {
+              border-color: @accent-color;
+              box-shadow: none;
+            }
+          }
+
+          button.btn.btn-primary {
+            background-color: @accent-color;
+            transition: all linear 0.15s;
+          }
+        }
+      }
+    }
+
+    .navbar-nav .nav-item a {
+      color: @text !important;
+
+      &:hover {
+        color: @subtext0 !important;
+      }
+      span {
+        color: @text !important;
+
+        &:hover {
+          color: @subtext0 !important;
+        }
+      }
+    }
+
+    .card {
+      background-color: @surface0 !important;
+      color: @subtext0 !important;
+      --bs-card-bg: @base !important;
+
+      .card-header {
+        border-color: @surface0 !important;
+      }
+    }
+
+    /* skin display link */
+    div.card-header {
+      span {
+        color: @text;
+
+        &:hover {
+          color: @accent-color !important;
+        }
+      }
+
+      strong a {
+        color: @accent-color !important;
+      }
+    }
+
+    /* skin display background */
+    .checkered {
+      background-image: none !important;
+      background-color: @mantle !important;
+    }
+
+    table.server-table {
+      > tbody > tr > td > a {
+        color: @accent-color !important;
+      }
+    }
```

Comment:
Are you sure there isn't a better selector for this? No class, ID, inline style, attribute, etc? This is unclear what it themes.